db/seeds.rb

```diff
@@ -1,12 +1,14 @@
 if Rails.env.development?
```

Comment:
> We need to change the line above as the preview servers run in production env. Not sure if we need to add an additional ENV variable to prevent this from being run on production servers.

I'd rather that be something we run/set manually for this task. We don't want someone to run it accidentally in production.